tfhe/src/shortint/key_switching_key/mod.rs

```diff
@@ -883,3 +884,25 @@ impl CompressedKeySwitchingKey {
         }
     }
 }
+
+pub struct KeySwitchingKeyConformanceParams2 {
```

Comment:
This could also benefit from a more explicit name such as `ShortintKeySwitchingKeyConformanceParams` or `KeySwitchingKeyMaterialConformanceParams`.

Reply:
Done (renamed `KeySwitchingKeyConformanceParams`)